src/lib/datepicker/calendar.html

```diff
@@ -9,6 +9,7 @@
       [dateFilter]="dateFilter"
       [maxDate]="maxDate"
       [minDate]="minDate"
+      [dateClass]="dateClass"
```

Comment:
Note that I've only set this up for the month view, because I wasn't sure whether it makes sense for the others. In the year or multi-year views we'd be calling the function with the first day of the month/year which would be a little ambiguous.